com.hack_computer.OperatingSystemApp.json

```diff
@@ -0,0 +1,55 @@
+{
+  "app-id": "com.hack_computer.OperatingSystemApp",
+  "add-extensions": {
+      "com.hack_computer.Clippy.Extension": {
+          "version": "beta",
+          "directory": "clippy",
+          "no-autodownload": false,
+          "autodelete": true
+      }
+  },
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "3.36",
+  "sdk": "org.gnome.Sdk",
+  "command": "com.hack_computer.OperatingSystemApp",
+  "finish-args": [
+    "--share=ipc",
+    "--socket=fallback-x11",
+    "--socket=wayland",
+    "--socket=pulseaudio",
+    "--device=dri",
+    "--own-name=com.hack_computer.OperatingSystemApp",
```

Comment:
you shouldn't need to own your own name, right?

Reply:
Right, I've removed that.